kernel/Cargo.toml

```diff
@@ -36,6 +36,7 @@ acpi = { git = "https://git.mirrors.dragonos.org/DragonOS-Community/acpi-rs.git"
 intertrait = { path = "src/libs/intertrait" }
 linkme = "0.2"
 ida = { path = "src/libs/ida" }
+mini-backtrace = { path = "/home/yaozhao/github/mini-backtrace", version = "0.1.4" }
```

Comment:
need to add a compilation flag to mini-backtrace to get rid of the stack protection in its build.rs. Is there an easy way to do it here?